app/views/provider_interface/courses/providers/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :browser_title, title_with_error_prefix(t('.title'), @wizard.errors.any?) %>
+
```

Comment:
do we have a back link here in the design?

Reply:
So this was intentional – the decision was to hold off adding bank links until the final wizard is in place and then will implement them for the whole flow